### PR TITLE
Outsiders lake trader home

### PR DESCRIPTION
**Pull request by faefux**
```SUMMARY: Content "fisherman traders home for outsiders faction"```

#### Purpose of change
Another building for outsiders faction 
#### Describe the solution
NA

#### Describe alternatives you've considered
NA

#### Additional context
Context: An NPC sits at the end of the pier in the armchair.  Part of a quest in which you visit him to make a deal for dried fish, eventually he does well with his fish business and branches off to selling more weed and moonshine with your help, eventually will give you an offer to become a salesman for him which will involve going to other factions and small groups and cutting deals with them.  Rewards and trading goods are be fish, weed, moonshine and high explosives and their reagents.

Didn't add specials because it's not strictly needed right now. Map shouldn't load.

### CODE REVIEW

File: data/json/mapgen/outsiders/fisherman_lake_home [hunk 1]
```diff
@@ -0,0 +1,314 @@
+[
```
Comment (anothersimulacrum) [inline data/json/mapgen/outsiders/fisherman_lake_home]: This isn't a `.json` file.

File: data/json/mapgen/outsiders/fisherman_lake_home [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+        { "item": "jar_glass", "x": 12, "y": 13, "repeat": 2 },
+        { "group": "misc_smoking", "x": 12, "y": 13 },
+        { "item": "joint", "x": 12, "y": 13 },
+        { "item": "joint_roach", "x": 12, "y": 13, "repeat": [ 2, 6 ] },
+        { "group": "hatstore_hats", "x": 11, "y": 13, "damage-min": 1, "damage-max": 3 },
+        { "item": "dress_wedding", "x": 11, "y": 13, "damage-min": 3, "damage-max": 4 },
+        { "group": "clothing_work_glasses", "x": 11, "y": 13 }
+      ],
+      "place_vehicles": [
+        { "vehicle": "fisher_canoe", "x": 20, "y": 16, "rotation": 90, "chance": 100 },
```
Comment (anothersimulacrum) [inline data/json/mapgen/outsiders/fisherman_lake_home]: This doesn't exist.

File: data/json/mapgen/outsiders/fisherman_lake_home [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+        { "vehicle": "inflatable_boat", "x": 10, "y": 19, "status": 0, "chance": 100 },
+        { "vehicle": "canoe", "x": 4, "y": 8, "rotation": 90, "status": 1, "chance": 100 },
+        { "vehicle": "wheelchair", "x": 12, "y": 14, "status": -1, "chance": 100 }
+      ],
+      "place_monsters": [
+        { "monster": "GROUP_FISH", "x": [ 16, 23 ], "y": [ 16, 23 ], "chance": 100, "repeat": 8 },
+        { "monster": "GROUP_FISH", "x": [ 0, 7 ], "y": [ 17, 23 ], "chance": 100, "repeat": 8 }
+      ],
+      "place_npcs": [ { "class": "NPC_OUTS_fisherman", "x": 16, "y": 22 } ]
```
Comment (anothersimulacrum) [inline data/json/mapgen/outsiders/fisherman_lake_home]: Nor does this NPC.

File: data/json/mapgen/outsiders/fisherman_lake_home [hunk 4]
```diff
@@ -0,0 +1,314 @@
...
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "outs_lake_shack_basement" ],
+    "weight": 100,
+    "object": {
+      "faction_owner": [ { "id": "outsiders", "x": [ 8, 4 ], "y": [ 18, 17 ] } ],
```
Comment (anothersimulacrum) [inline data/json/mapgen/outsiders/fisherman_lake_home]: This faction doesn't exist

File: data/json/mapgen/outsiders/fisherman_lake_home [hunk 5]
```diff
@@ -0,0 +1,314 @@
...
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        "
+      ],
+      "faction_owner": [ { "id": "outsiders", "x": [ 7, 23 ], "y": [ 7, 21 ] } ],
```
Comment (anothersimulacrum) [inline data/json/mapgen/outsiders/fisherman_lake_home]: Ditto

File: data/json/mapgen/outsiders/fisherman_lake_home [hunk 6]
```diff
@@ -0,0 +1,314 @@
...
+        "       8=&&````x=       ",
+        "        ====,,,==       ",
+        "            S88         ",
+        "            888         ",
+        "            888         ",
+        "          8888888       ",
+        "          888888H       ",
+        "                        "
+      ],
+      "faction_owner": [ { "id": "outsiders", "x": [ 7, 23 ], "y": [ 7, 23 ] } ],
```
Comment (anothersimulacrum) [inline data/json/mapgen/outsiders/fisherman_lake_home]: ditto

File: data/json/mapgen/outsiders/fisherman_lake_home (hunk 4 above)
Comment (anothersimulacrum) [inline data/json/mapgen/outsiders/fisherman_lake_home]:
```suggestion
      "faction_owner": [ { "id": "outsiders", "x": [ 8, 4 ], "y": [ 18, 17 ] } ],
      "fill_ter": "t_rock",
```

File: data/json/mapgen/outsiders/fisherman_lake_home [hunk 7]
```diff
@@ -0,0 +1,314 @@
...
+      ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "outs_lake_shack_roof" ],
+    "weight": 100,
+    "object": {
+      "fill_ter": "t_rock",
```
Comment (anothersimulacrum) [inline data/json/mapgen/outsiders/fisherman_lake_home]:
```suggestion
      "fill_ter": "t_open_air",
```

**Comment by curstwist** [comment]
glad to see you are still working on this project!

**Comment by faefux** [comment]
Aghhh, upd the wrong version of this file, moving files across machines and dealing with this site stressed me out and I make mistakes. Sorry. 

@curstwist yea, I have quite a lot at this point so I'm in the process of slimming some files down to start uploading.

**Comment by faefux** [comment]
Aghhh, upd the wrong version of this file, moving files across machines and dealing with this site stressed me out and I make mistakes. Sorry. 

@curstwist yea, I have quite a lot at this point so I'm in the process of slimming some files down to start uploading.

**Pull request closed without merging**